script-library/docker-in-docker-debian.sh

```diff
@@ -325,6 +326,10 @@ tee /usr/local/share/docker-init.sh > /dev/null \
 
 set -e
 
+AZURE_DNS_OVERRIDE=$AZURE_DNS_OVERRIDE
+EOF
+
+tee -a ./docker-init.sh > /dev/null << 'EOF'
```

Comment:
nit: Could we keep the style of this `tee` command the same as the one above (same line breaks, using the full `/usr/local/share/docker-init.sh` file path)